docker/migrations/docker-entrypoint.sh

```diff
@@ -22,4 +22,7 @@ php bin/console doctrine:migrations:migrate --no-interaction
 
 if [ "$LOAD_FIXTURES" = "1" ]; then
     php bin/console sylius:fixtures:load --no-interaction
+
+    # make the image files created by fixtures accessible by fpm which runs with user www-data
+    chown -R www-data:www-data public/media/image
```

Comment:
@Ferror this will fix the issue, but it's just like applying FlexTape... ideally we should adopt non-root containers and make GIDs and UIDs configurable via build args and env vars, like this: https://github.com/laradock/laradock/blob/96e0f2e92f6e5b128a71cc77a05b297ac4a0eadb/workspace/Dockerfile#L43-L47

Reply:
I realized that if I launched the fixtures manually from the php container, I had to reset the rights each time afterwards.

```
php bin/console sylius:fixtures:load -n && chown -R www-data:www-data public/media/image
```